test/parallel/test-experimental-shared-value-conveyor.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+// Flags: --harmony-struct
+
+const common = require('../common');
+const assert = require('assert');
+const { Worker, isMainThread, parentPort } = require('worker_threads');
+
+if (isMainThread) {
```

Comment:
We have a CI job that checks if tests can be run inside a worker so this would not work in that job: https://ci.nodejs.org/job/node-test-commit-custom-suites-freestyle/30713. This can be replaced with something like:

```suggestion
// Do not use isMainThread so that this test itself can be run inside a Worker.
if (!process.env.HAS_STARTED_WORKER) {
  process.env.HAS_STARTED_WORKER = 1;
```
That we've been doing in other tests.

Reply:
Thanks, applied.